apps/dashboard/src/components/conditions-editor/add-group-action.tsx

```diff
@@ -0,0 +1,24 @@
+import { ActionWithRulesAndAddersProps } from 'react-querybuilder';
+
+import { StackedPlusLine } from '@/components/icons/stacked-plus-line';
+import { Button } from '@/components/primitives/button';
+
+export const AddGroupAction = ({ label, title, level, rules, handleOnClick }: ActionWithRulesAndAddersProps) => {
```

Comment:
This represents the add group button in the editor UI. The button will be hidden if there are 10 conditions applied or the group is nested (restriction from the definition of done).

![Screenshot 2025-01-14 at 12 20 00](https://github.com/user-attachments/assets/4463498d-90d2-46fd-bc33-e0f2a6eb2768)